fk/teamjoin.go

```diff
@@ -69,6 +75,27 @@ func teamJoin(teamUUID uuid.UUID) exitCode {
 
 }
 
+func ensureNoExistingRequests(teamUUID uuid.UUID) exitCode {
+	existingRequests, err := db.GetRequestsToJoinTeam(teamUUID)
+	if err != nil {
+		out.Print(ui.FormatFailure("Failed to check for existing requests", nil, err))
+		return 1
+	}
+	if len(existingRequests) > 0 {
+		out.Print(ui.FormatWarning(
+			"You've already requested to join "+existingRequests[0].TeamName,
+			[]string{
+				"You requested to join " + existingRequests[0].TeamName + " " +
```

Comment:
be great if you could use the helper function I've made for this, I'll put the commit as a separate pull request

Reply:
Wilco!

Reply:
Done!